Python/bigtable/read_bigtable.py

```diff
@@ -0,0 +1,54 @@
+import logging
+import apache_beam as beam
+
+from apache_beam import Map
+from apache_beam.io.gcp.bigtableio import ReadFromBigtable
+from apache_beam.options.pipeline_options import PipelineOptions
+
+
+def run():
+    """
+    This Apache Beam pipeline retrieves data from Google Cloud Bigtable table.
+    It performs the following steps:
+    1. Read from Bigtable:
+        - The pipeline reads data from a specified Bigtable instance and table.
```

Comment:
I also think we have to use correct product name "Cloud Bigtable" where possible